apps/sms/locales/sms.en-US.properties

```diff
@@ -308,6 +349,19 @@ message-delivery-status-delivered.ariaLabel          = Delivered
 message-delivery-status-read.ariaLabel               = Read
 message-delivery-status-error.ariaLabel              = Error
 
+# Late arrival notices
+late-notice-label.ariaLabel           = Late arrival
+late-arrival-notice-in-months         = {[ plural(lateness) ]}
+late-arrival-notice-in-months[zero]   = {{ lateness }}mth late
+late-arrival-notice-in-months[one]    = {{ lateness }}mth late
+late-arrival-notice-in-months[two]    = {{ lateness }}mths late
+late-arrival-notice-in-months[few]    = {{ lateness }}mths late
+late-arrival-notice-in-months[many]   = {{ lateness }}mths late
+late-arrival-notice-in-months[other]  = {{ lateness }}mths late
+late-arrival-notice-in-days           = {{ lateness }}d late
+late-arrival-notice-in-hours          = {{ lateness }}h late
+late-arrival-notice-in-minutes        = {{ lateness }}m late
```

Comment:
in our case I think it's ok, especially because we'll likely never get months for this value. But having "Late arrival" all the time is too long. This string ends up at the bottom of a message in the conversation.

So Zibi, flod, do you prefer to have 'months' without an abbreviation here? Does it make more sense? Or should we have an abbreviation for the locales that can have one (we can add a localization note to state this of course) ?